public/schemas/searchFilters/announcements-search.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "filterCreationSchema",
+  "listingType": "announcements",
+  "items": [
+    {
+      "title": {
+        "id": "schema.forSale.price",
```

Comment:
It is a bit confusing reading this file and seeing those references to strings from the UI JSON schemas that are not of the same type as the filter. Since the filters apply to fields that are collected during listing creation, there should not be any reason to not use those ?
e.g. for the "housing" filter, use strings from the [housing UI schema](https://github.com/OriginProtocol/origin-dapp/blob/master/public/schemas/housing.json). for the "announcement" filters, use strings from the [announcement UI schema](https://github.com/OriginProtocol/origin-dapp/blob/master/public/schemas/announcements.json) , etc...

Reply:
Yes that makes sense. We will need to be careful of that when creating the real filters and not just these ones that are intended for testing UI. 

Have changed the strings in the `announcements-search.json` so that we do not forget about that in the future.